.gitignore

```diff
@@ -1,2 +1,5 @@
 node_modules
 package-lock.json
+favicon.ico
+$__plugins_felixhayashi_tiddlymap_misc_defaultViewHolder.tid
```

Comment:
FYI, there are _probably_ more than this, TiddlyWiki has some funny behaviour with timestamps. Not worth doing anything about it yet though.

Reply:
I'm not sure about the timestamps. It's a bit awkward that were using a version control system *and* use timestamps in the files.

Reply:
Yes it is. Hopefully we can hide some of that in the interim. When Pete and I talked about this I raised that if you dig into fixing wiki editing too much it starts to dovetail with how the website works, and then into branding and UX and a whole bunch of other stuff that is not as important yet, so a stop-gap seems appropriate for now.